ui/src/views/Quickstart/index.jsx

```diff
@@ -29,16 +29,15 @@ import githubQuery from './github.graphql';
 
 const initialYaml = {
   version: 1,
-  tasks: [
-    {
+  policy: {
+    pullRequests: '',
+  },
+  tasks: {
+    $match: {
+      taskId: { $eval: 'as_slugid("pr_task")' },
```

Comment:
For some reason this line and several lines below it do not show up for me on the deployed website. If you run a local build while developing, it will be much easier to see these issues and to debug them.